tensorboard/uploader/exporter.py

```diff
@@ -276,8 +425,8 @@ def _mkdir_p(path):
             raise
 
 
-def _open_excl(path):
-    """Like `open(path, "x")`, but Python 2-compatible."""
+def _open_excl(path, mode="w"):
+    """Like `open(path, "x" + mode)`, but Python 2-compatible."""
```

Comment:
nit: This is cute :-) but `xw` is not a valid mode; `x` and `xb` are, so
maybe `open(path, mode.replace("w", "x"))` does the job instead?

Reply:
Done.